### PR TITLE
524 Normalize the media queries importing from unique .js (PR to develop)

### PR DESCRIPTION
**Pull request by N3BB3Z4R**
Changed this files to normalize mediaqueries from unique mediaQueries.js :

// DONE /components /composed /AdCardList /AdCardList.styles.js
// DONE /components /composed /Card /Card.style.js
// DONE /components /composed /Charts /BarChart /BarGraphic.styles.js
// DONE /components /composed /Charts /LineChart /LineChart.styles.js
// DONE /components /composed /Gallery /Gallery.styles.js
// DONE /components /composed /GlobalFilters /GlobalFilters.styles.js
// DONE /components /composed /SearchBar /SearchBarStyles.js
// DONE /components /layout /Footer /Footer.styles.js
// DONE /components /units /inputNumber /InputNumber.styles.js
// DONE /components /units /Notifications /Notifications.styles.js
// DONE /components /units /SearchButton /SearchButtonStyles.js
// DONE /screens /Ad /Ad.styles.js
// DONE /screens /AdList /AdCard /AdCard.style.js
// DONE /screens /AdList /AdListFilter /AdListFilter.style.js
// DONE /screens /AdList /AdList.style.js
// DONE /screens /CreateNewAd /CreateNewAd.styles.js
// DONE /screens /Dashboard /Dashboard.style.js
// DONE /screens /EditAd /EditAd.styles.js
// DONE /screens /ListaUsuariosAdmins /ListaUsuariosAdmins.styles.js
// DONE /screens /MyBills /Bill.styles.js
// DONE /screens /MyBills /MyBills.styles.js
// DONE /screens /Search /Search.style.js
// DONE /screens /UserFlow /Profile /Profile.styles.js

Warning: As it can see in the mediaQueries.js I've encountered too much numerical variations, I managed all to avoid to break actual template nor study case by case, but its recommendable to fix those styles.

### CODE REVIEW

File: frontend/src/theme/mediaQueries.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+// DONE /screens /AdList /AdListFilter /AdListFilter.style.js
+// DONE /screens /AdList /AdList.style.js
+// DONE /screens /CreateNewAd /CreateNewAd.styles.js
+// DONE /screens /Dashboard /Dashboard.style.js
+// DONE /screens /EditAd /EditAd.styles.js
+// DONE /screens /ListaUsuariosAdmins /ListaUsuariosAdmins.styles.js
+// DONE /screens /MyBills /Bill.styles.js
+// DONE /screens /MyBills /MyBills.styles.js
+// DONE /screens /Search /Search.style.js
+// DONE /screens /UserFlow /Profile /Profile.styles.js
```
Comment (kevinmamaqi) [inline frontend/src/theme/mediaQueries.js:32]: Remove the comments please, they don't add value to next developers.

File: frontend/src/theme/mediaQueries.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+// Screen Sizes
+const responsiveSizes = {
```
Comment (kevinmamaqi) [inline frontend/src/theme/mediaQueries.js:35]: Simplify to have less sizes: https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/

File: frontend/src/theme/mediaQueries.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+	// Custom min-to-max-width selectors
+	between481and768: `screen and (min-width: 481px) and (max-width: 768px)`,
+	between768and1024: `screen and (min-width: 768px) and (max-width: 1024px)`,
+	between768and1400: `screen and (min-width: 768px) and (max-width: 1400px)`,
+};
+
+// Generic MediaQuery Values
+export const Values = {
+	fontSize14: `font-size: 14px;`,
```
Comment (kevinmamaqi) [inline frontend/src/theme/mediaQueries.js:106]: This I don't think it's necessary.

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]
@N3BB3Z4R por favor, revisa los cambios.

File: frontend/src/theme/mediaQueries.js (hunk 1 above)
Comment (N3BB3Z4R) [reply frontend/src/theme/mediaQueries.js:32]: Ok. I'll delete it.

File: frontend/src/theme/mediaQueries.js (hunk 2 above)
Comment (N3BB3Z4R) [reply frontend/src/theme/mediaQueries.js:35]: Understood. But what happens with customs? I should override the custom uses along the project and study case by case? I think in some situations custom number mediaqueries were used to fix bad templating behaviors.

File: frontend/src/theme/mediaQueries.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+	laptopLarge: "1440px",
+	desktopLarge: "2560px",
+};
+
+// TODO Fix and Unify the CUSTOM max and min Device Values looking case by case
+
+// Device MediaQuery Selectors
+export const Device = {
+	// Standard min-width device selectors
+	minMobileSmall: `(min-width: ${responsiveSizes.mobileSmall})`,
```
[the remark this answers is not in the document]
Comment (N3BB3Z4R) [reply frontend/src/theme/mediaQueries.js:52]: Understood.

File: frontend/src/theme/mediaQueries.js (hunk 3 above)
Comment (N3BB3Z4R) [reply frontend/src/theme/mediaQueries.js:106]: Ok, I'll delete this too.

**Comment by N3BB3Z4R** [comment]
fix district & map template
fix mediaqueries in whole project
fix minor css errors

**Comment by N3BB3Z4R** [comment]
Normalize mediaquery uses and make mobile first in whole project.
Fix and improve css in several templates.

**Review by kevinmamaqi — APPROVED** [review]

**Pull request merged**